inc/commonitilobject.class.php

```diff
@@ -8203,4 +8203,19 @@ public function getLinkedItems(bool $addNames = true) :array {
 
       return $assets;
    }
+
+   /**
+    * Should impact tab be displayed? Check if there is a valid linked item
+    *
+    * @return boolean
+    */
+   protected function showImpactTab() {
```

Comment:
hasImpactTab ?

Reply:
or maybe directly addImpactTab which do the checks and call directly addStandardTab

Reply:
Since this is really specific for ITIL, I guess commonitil is not wrong